Makefile

```
@@ -1,18 +1,29 @@
 RUSTUP = rustup
+
 TOOLCHAIN = $(shell cat rust-toolchain)
-CARGO = $(RUSTUP) run --install $(TOOLCHAIN) cargo --color always
+CARGO = $(RUSTUP) run $(TOOLCHAIN) --install cargo --color always
+
+NIGHTLY_TOOLCHAIN = "nightly-2019-07-31"
+CARGO_NIGHTLY = $(RUSTUP) run $(NIGHTLY_TOOLCHAIN) --install cargo --color always
```

```suggestion
CARGO_NIGHTLY = $(RUSTUP) run --install $(NIGHTLY_TOOLCHAIN) cargo --color always
```
See previous comment.